pkg/api/handlers/generic/config.go

```diff
@@ -0,0 +1,8 @@
+package generic
+
+type CtrCreateResponse struct {
```

Comment:
Please comment exported types. The linter should complain about it.

Reply:
s/CtrCreateResponse/CreateContainerResponse/ to make the type match the identifier of the function.